titus-server-master/src/main/java/com/netflix/titus/master/kubernetes/pod/v0/V0SpecPodFactory.java

```diff
@@ -177,6 +182,29 @@ public V1Pod buildV1Pod(Job<?> job, Task task, boolean useKubeScheduler, boolean
         return new V1Pod().metadata(metadata).spec(spec);
     }
 
+    private List<V1Volume> buildV1Volumes(List<Volume> volumes) {
+        if (volumes == null) { return null;}
```

Comment:
It would be safer to return `Collections.emtpyList()`

Reply:
Oh right